### ISSUE TITLE
Label Import Issue

### ISSUE THREAD

**Issue by trasse** [yapic/yapic#8]
Dear Christoph,

I have trouble testing Yapic.
I fail to import the label from "leaf_labels.ilp"
The file itself is fine (in Ilastik i see the label).

I run your code from inside a Jupyter Notebook (should not be an issue, or?)

This line:
t.load_training_data(img_path, label_path)
gives me this error:

.../anaconda3/envs/ml3_cvk_YP/lib/python3.6/site-packages/pyilastik/ilastik_storage_version_01.py:226: UserWarning: No labels found in Ilastik file - cannot approximate image size
  warnings.warn(msg)

Please advise!

I have:

RTX 2080 (8 GB RAM)
Ubuntu 18.04
Python 3.6 

**
Tensorflow_GPU 1.12.0
pyilastik 0.0.5
cudatoolkit 9,2
cudnn 7.3.1
keras 2.2.4
**

(Works fine for everything else...)

Thanks a lot and Kind regards

Tobias

**Comment by cmohl2013** [comment]
Dear Tobias,

thank you for the bug report. We have never tested on Jupyter Notebook. I'll try to import the labels on our test system (not Anaconda, not Jupyter, plain official Python 3.6 distribution) and give feedback if it works as expected or not.

**Comment by cmohl2013** [comment]
Dear Tobias,

I reproduced your bug on our test system, so it is not due to Jupyter installation. We would need a test for the label import of this tutorial data. And maybe update the tutorial data file.

**Comment by cmohl2013** [comment]
we have uploaded a new leaf labels test dataset, produced with Ilastik version 1.3.3
